src/tilda.c

```diff
@@ -746,11 +764,24 @@ int main (int argc, char *argv[])
         if (atol (getenv ("VTE_PROFILE_MEMORY")) != 0)
             g_mem_set_vtable (glib_mem_profiler_table);
 #endif
+    /* Parse the command line */
+    config_file = NULL;
+    need_wizard = parse_cli (argc, argv, &config_file);
+
+    if (config_file != NULL) {
+        if (!g_file_test (config_file, G_FILE_TEST_EXISTS)) {
+            g_printerr (_("Specified config file '%s' does not exist. Reverting to default path.\n"),
+                    config_file);
+            config_file = NULL;
+        }
+    }
+    if (config_file == NULL) {
+        config_file = get_config_file_name (lock.instance);
+    }
+
     /* Start up the configuration system */
     gint config_init_result = config_init (config_file);
 
-    /* Parse the command line */
-    need_wizard = parse_cli (argc, argv);
```

Comment:
Thanks for your suggestion @pik, this should be fixed now.
(Note to self: next time, I'll push a new commit instead of amending the last one...)